src/icons/newline.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" fill="currentColor"><path fill-rule="evenodd" clip-rule="evenodd" d="M12 5V6.984C12 7.11661 11.9473 7.24379 11.8535 7.33755C11.7598 7.43132 11.6326 7.484 11.5 7.484H4.618L6.251 5.851L5.544 5.144L3.423 7.265L3 7.688V8.256L5.544 10.8L6.251 10.093L4.641 8.484H11.5C11.8978 8.484 12.2793 8.32597 12.5607 8.04466C12.842 7.76336 13 7.38182 13 6.984V5H12Z"/></svg>
```

Comment:
Even though you likely copied this one from the "no-newline" icon, it would be great if this was aligned to the pixel grid to avoid pixelation:

<img width="563" alt="CleanShot 2021-11-04 at 11 26 25@2x" src="https://user-images.githubusercontent.com/35271042/140398168-52556fa0-6231-4b79-ba60-ea9a863bc6b0.png">

<img width="572" alt="CleanShot 2021-11-04 at 11 26 41@2x" src="https://user-images.githubusercontent.com/35271042/140398210-454f081c-ca5c-472e-90bb-0c98f98711d0.png">

Suggestion to align it:

```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" fill="currentColor"><path fill-rule="evenodd" clip-rule="evenodd" d="M12 5.5V7.484C12 7.61661 11.9473 7.74379 11.8535 7.83755C11.7598 7.93132 11.6326 7.984 11.5 7.984H4.618L6.251 6.351L5.544 5.644L3.423 7.765L3 8.188V8.756L5.544 11.3L6.251 10.593L4.641 8.984H11.5C11.8978 8.984 12.2793 8.82597 12.5607 8.54466C12.842 8.26336 13 7.88182 13 7.484V5.5H12Z"/></svg>
```

<img width="564" alt="CleanShot 2021-11-04 at 11 27 02@2x" src="https://user-images.githubusercontent.com/35271042/140398269-ba9e4417-8242-4f0c-af07-f75a18ed6446.png">


<img width="556" alt="CleanShot 2021-11-04 at 11 26 56@2x" src="https://user-images.githubusercontent.com/35271042/140398249-3e7319e4-5546-4bff-a7a4-811aadd75c59.png">

Reply:
I have used figma and just moved original icon to right, removed circle and extended a little, to make icon longer.
![image](https://user-images.githubusercontent.com/8218016/140405943-af150d1c-ddbc-4f9e-a418-ae33c857e442.png)

Reply:
yea if you can just align it to the grid that would be great, I'll also update the original `no-newline` to reflect the same thing